.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Provide a general summary of the issue in the Title above -->
```

Comment:
The template will be automatically loaded when someone opens a new Issue/PR, there is no copy/paste involved 
https://github.com/blog/2111-issue-and-pull-request-templates